bionty/models.py

```diff
@@ -276,7 +276,7 @@ def import_source(
         """
         from .core._add_ontology import add_ontology_from_df, check_source_in_db
 
-        if hasattr(cls, "ontology_id"):
+        if hasattr(cls, "ontology_id") or hasattr(cls, "_name_field"):
```

Comment:
This is yet another fall back that is probably not necessary but it should not hurt either.

Reply:
See the next comment.

Reply:
Ahh actually, I misremembered. This is needed for `CellMarker` because it does not have a `ontology_id` field.